src/tabs/home/Home.js

```diff
@@ -98,17 +97,20 @@ class Home extends PureComponent {
                         );
 
                         return (
-                          <TouchableScale
-                            onPress={() =>
-                              this.handleOnPress({ id: featuredItem.id })
-                            }
-                          >
-                            <ContentCardConnected
-                              Component={BrandedCard}
-                              contentId={featuredItem.id}
-                              isLoading={isFeaturedLoading}
-                            />
-                          </TouchableScale>
+                          <>
+                            <LiveButton contentId={featuredItem.id} />
```

Comment:
Just a question here: Why does the `LiveButton` need the `contentId`? Seems to me the `LiveConsumer` should just check to see whether we're live or not and it doesn't need to know what item we're looking at, right?

Reply:
cores version does, and since we're using there's now, I matched the implementation. Ours was custom before.

Reply:
I guess if we need it, we need it. It's extra data we're passing around for no reason, but I don't see a way to NOT do it. Resolved.